src/sentry/static/sentry/app/views/stream.jsx

```diff
@@ -623,17 +638,30 @@ const Stream = React.createClass({
   renderAwaitingEvents() {
     let org = this.getOrganization();
     let project = this.getProject();
+    let platformName = project.platform
+      ? getPlatformName(project.platform)
+      : 'JavaScript';
     let sampleLink = null;
     if (this.state.groupIds.length > 0) {
       let sampleIssueId = this.state.groupIds[0];
+
       sampleLink = (
         <p>
           <Link to={`/${org.slug}/${project.slug}/issues/${sampleIssueId}/?sample`}>
-            {t('Or see a sample Javascript event')}
+            {tct('Or see your sample %s event', platformName)}
```

Comment:
based on percy, i think you need to use `[variable]` for formatting with `tct`, also i might have been wrong and maybe `t` would've been fine. I thought `tct` was the one that supported interpolation, but it might just be better if you have multiple variables/are using react components